doajtest/unit/test_api_bulk_application.py

```diff
@@ -15,6 +18,7 @@ def tearDown(self):
     def test_01_create_applications_success(self):
         # set up all the bits we need - 10 applications
         data = ApplicationFixtureFactory.incoming_application()
+        del data["admin"]["current_journal"]
```

Comment:
It could be, but I've explicitly ruled out messing with the fixture generation in this project.  We have got a line item in the refactor to revisit the fixtures, so I want to do a full treatment of them there.  Otherwise, anything I did here would be half a job, and might turn out to be wasted effort.